test/opt/local_access_chain_convert_test.cpp

```diff
@@ -96,6 +96,208 @@ OpFunctionEnd
                                                      true);
 }
 
+TEST_F(LocalAccessChainConvertTest, DebugInfo) {
```

Comment:
If would be nice if these tests told me what you where actually looking for.  A name like "DebugValueCorrectlyUpdated" or "DontOptimizeDebugDeclare" with a few comment are good.

Reply:
updated